src/optionList.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
How are you generating this list? 
Are you reading it from the original repository o somewhere else? we need to make sure that if a new icon category is added in the main repo, then when pulling (or updating the dependency from this repo) the new version this list gets updated too

Reply:
This list would be generated by `arrange-icons.js` file. It can be regenerated everything the icons package is updated, and we push a new release.

Reply:
Related to the previous comment.